manifests/c/ClashVergeRev/ClashVergeRev/2.1.0/ClashVergeRev.ClashVergeRev.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with WinGet Releaser using komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ClashVergeRev.ClashVergeRev
+PackageVersion: 2.1.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- clash
+ProductCode: Clash Verge
+ReleaseDate: 2025-02-26
+AppsAndFeaturesEntries:
+- ProductCode: Clash Verge
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v2.1.0/Clash.Verge_2.1.0_x64-setup.exe
+  InstallerSha256: EAA3BCB212D22A17B9CD58A51281B93C5EC72490485C3719E0425FD6004A6FEE

```

```suggestion
  InstallerSha256: 7D55C4CBD39343CD79183A98272E20AE9B8600CF577E8DCB73101012EE68D48F
```

(Automated response - build 982.)